projects/core/directives/textfield-controller/textfield.controller.ts

```diff
@@ -34,7 +34,7 @@ export class TuiTextfieldController {
     }
 
     get customContent(): PolymorpheusContent {
-        return this.customContentDirective.customContent || ``;
+        return this.customContentDirective.customContent ?? ``;
```

Comment:
Is this fallback necessary at all? Maybe it was before we added `null` to `PolymorpheusContent`?